addons/website/static/src/less/website.ui.less

```diff
@@ -12,6 +12,8 @@ body.o_connected_user {
     .o-position-absolute(0, 0, auto, 0);
     position: fixed;
     z-index: 1001;
+    border: none;
```

Comment:
👍  Yes I agree. That border is in place since the v9 restyling, but it's not supposed to be there since we changed the color for v10.